company_linkage/sql/workforce_visualization_data.sql

```diff
@@ -0,0 +1,21 @@
+  -- Selecting the companies we want to leave out
+WITH
```

Comment:
Ah! I can see how the naming convention here would be confusing. Perhaps I can clear it up with comments? Basically, what is going on here, though, is:

1. In visualization_data, I am running a rule-based omission, and eliminating any orgs, and their corresponding CSET_ids, that break my set of rules
2. I am saving the *qualifying* (not omitted) results in `visualization_data_omit_by_rule`. That is, that table is the result of running my rule-based omission, but contains the non-omitted data.
3. Then, for later tables like this one, I'm checking all my CSET ids against that table to see which ids aren't in my "qualified orgs" table, and omitting any ids that aren't qualified.

I tried to clarify this by naming it "omit by rule" rather than, say "omitted" or "excluded" or something. But I admit that clarification was probably insufficient.